File: android/app/src/main/java/com/phicdy/totoanticipation/view/fragment/GameHistoryFragment.java
package com.phicdy.totoanticipation.view.fragment;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.webkit.WebResourceRequest;
import android.webkit.WebSettings;
import android.webkit.WebView;
import android.webkit.WebViewClient;

import com.phicdy.totoanticipation.R;
import com.phicdy.totoanticipation.model.GameHistory;

public class GameHistoryFragment extends Fragment {
    public static final String ARG_HOME_TEAM = "hogeTeam";
    public static final String ARG_AWAY_TEAM = "awayTeam";

    private WebView webView;
    private String homeTeam;
    private String awayTeam;

    public GameHistoryFragment() {
    }

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        if (getArguments().containsKey(ARG_HOME_TEAM) &&
                getArguments().containsKey(ARG_AWAY_TEAM)) {
            homeTeam = getArguments().getString(ARG_HOME_TEAM);
            awayTeam = getArguments().getString(ARG_AWAY_TEAM);
        }
    }

    @SuppressLint("SetJavaScriptEnabled")
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        View rootView = inflater.inflate(R.layout.item_detail, container, false);

        webView = ((WebView) rootView.findViewById(R.id.item_detail));
        WebSettings webSettings = webView.getSettings();
        webSettings.setJavaScriptEnabled(true);
        webView.setWebViewClient(new WebViewClient() {
            @Override
            public boolean shouldOverrideUrlLoading(WebView view, WebResourceRequest request) {
                return super.shouldOverrideUrlLoading(view, request);
            }
        });

        return rootView;
    }

    @Override
    public void onResume() {
        super.onResume();
        webView.loadUrl(new GameHistory().url(homeTeam, awayTeam));
    }
}

File: android/legacy/src/main/java/com/phicdy/totoanticipation/legacy/view/fragment/GameHistoryFragment.kt
package com.phicdy.totoanticipation.legacy.view.fragment

import android.annotation.SuppressLint
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.webkit.WebView
import android.webkit.WebViewClient
import androidx.fragment.app.Fragment
import com.phicdy.totoanticipation.legacy.R
import com.phicdy.totoanticipation.legacy.model.GameHistory

class GameHistoryFragment : Fragment() {

    private lateinit var webView: WebView
    private lateinit var homeTeam: String
    private lateinit var awayTeam: String

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        arguments?.let {
            if (it.containsKey(ARG_HOME_TEAM) && it.containsKey(ARG_AWAY_TEAM)) {
                homeTeam = it.getString(ARG_HOME_TEAM, "")
                awayTeam = it.getString(ARG_AWAY_TEAM, "")
            }
        }
    }

    @SuppressLint("SetJavaScriptEnabled")
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        val rootView = inflater.inflate(R.layout.item_detail, container, false)

        webView = rootView.findViewById<View>(R.id.item_detail) as WebView
        val webSettings = webView.settings
        webSettings.javaScriptEnabled = true
        webView.webViewClient = WebViewClient()
        return rootView
    }

    override fun onResume() {
        super.onResume()
        webView.loadUrl(GameHistory().url(homeTeam, awayTeam))
    }

    companion object {
        const val ARG_HOME_TEAM = "hogeTeam"
        const val ARG_AWAY_TEAM = "awayTeam"
    }
}
